platform/viewer/cypress/support/commands.js

```diff
@@ -30,12 +30,73 @@ Cypress.Commands.add('openStudy', (patientName) => {
     cy.visit('/');
     cy.get('#patientName')
       .type(patientName);
-    cy.get('.studylistStudy > .patientName')
-      .contains(patientName)
-      .click();
+
+    const patientResult = '.studylistStudy > .patientName';
+
+    cy.get(patientResult)
+      .then({ timeout: 5000 }, ($patientResult) => {
+        cy.contains(patientName)
+        .click();
+      })
   }
 );
 
 Cypress.Commands.add('drag', {prevSubject: 'element',},
   (...args) => DragSimulator.simulate(...args)
 );
+
+// Command to perform two clicks into two different positions. Each position must be (x, y).
+// The positions are considering the element as reference, therefore, top-left of the element will be (0, 0).
+Cypress.Commands.add('addLine', (element, initPosition, finalPosition) =>  {
+
+  cy.get(element)
+    .click(initPosition[0], initPosition[1], { force: true })
+    .then(() =>{
+      cy.get(element)
+        .trigger('mousemove', { clientX: finalPosition[0], clientY: finalPosition[1] })
+        .click({ force: true })
+    })
+});
+
+
+// Command to perform three clicks into three different positions. Each position must be (x, y).
+// The positions are considering the element as reference, therefore, top-left of the element will be (0, 0).
+Cypress.Commands.add('addAngle', (element, initPosition, midPosition, finalPosition) =>  {
+
+  cy.get(element)
+    .click(initPosition[0], initPosition[1], { force: true })
+    .then(() =>{
+      cy.get(element)
```

Comment:
You shouldn't need to "get" the element again. You can leverage the pattern shown in Cypress's closure docs to clean this up: https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Closures

Reply:
It's necessary to grab the element again. I replaced it with `cy.wrap()` which is more appropriate then `cy.get()` in this context.